core/src/main/java/io/fabric8/maven/core/util/ProfileUtil.java

```diff
@@ -62,6 +62,9 @@ public static Profile findProfile(String profileArg, File resourceDir) throws IO
             String profile = profileArg == null ? DEFAULT_PROFILE : profileArg;
             Profile profileFound = lookup(profile, resourceDir);
             if (profileFound != null) {
+                if(profileFound.getParentProfile() != null) {
+                    profileFound = inheritFromParentProfile(profileFound, resourceDir);
```

Comment:
Would be better to add log debug/info statement so that user can get visibility of extended profile is being used